tests/init_test.go

```diff
@@ -140,9 +140,15 @@ func (tm *testMatcher) whitelist(pattern string) {
 	tm.whitelistpat = regexp.MustCompile(pattern)
 }
 
-func TestDummyMatcherToKeepBothStaticcheckandHolimanHappy(t *testing.T) {
-	tm := testMatcher{}
-	tm.whitelist("")
+func TestWhitelist(t *testing.T) {
```

Comment:
Please move the test somewhere else in the file. While this is a test file, the code around here defines an API for use by other tests. Best to not dump a unit test right in the middle of that.